sycl/test-e2e/KernelFusion/lit.local.cfg

```diff
@@ -0,0 +1,5 @@
+if config.sycl_be in ['ext_oneapi_hip']:
+   config.unsupported = True
+
+if 'fusion' not in config.available_features:
+   config.unsupported = True
```

Comment:
Do you plan to use this approach for all directories? 

A potential downside IMO is that is not immediately visible from the test file itself what is required and on which backends this test/feature is unsupported.